### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by trideset3**
- Added configurable query parameter token to the layer provider

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,16 +114,16 @@ Under the `maps` section, map layers are associated with data provider layers an
...
 uri = "postgresql://tegola:<password>@localhost:5432/tegola?ssl_mode=prefer" # database connection string
 
   [[providers.layers]]
   name = "landuse"
   # MVT data provider must use SQL statements
   # this table uses "geom" for the geometry_fieldname and "gid" for the id_fieldname so they don't need to be configured
-  # Wrapping the geom with ST_AsMVTGeom is required. 
-  sql = "SELECT ST_AsMVTGeom(geom,!BBOX!) AS geom, gid FROM gis.landuse WHERE geom && !BBOX!"
+  # Wrapping the geom with ST_AsMVTGeom is required.
+  sql = "SELECT ST_AsMVTGeom(geom,!BBOX!) AS geom, gid FROM gis.landuse WHERE geom && !BBOX! !PARAM!"
```
Comment (ARolek) [inline README.md:126]: What do you think about carving the `!PARAM!` token into a separate example? That way we can introduce the concept to the user after they have the `!BBOX!` token working.

Comment (trideset3) [reply README.md:126]: You are right, that makes sense. I'll add another commit.

**Review by ARolek — APPROVED** [review]
LGTM. Please squash commits into a single commit.

File: README.md (hunk 1 above)
Comment (trideset3) [reply README.md:128]: No worries, I hope this will be ok.

Comment (trideset3) [reply README.md:128]: @ARolek I forgot the --force and merge was also commited. Please let me know if I should squash this again.

**Comment by ARolek** [comment]
@trideset3 this has been merged in. I pulled down your branch and got everything into a single commit prior to merging. Just letting you your changes did land. Thanks!

**Pull request closed without merging**